docs/installation-guide/src/main/asciidoc/installing.adoc

```diff
@@ -660,81 +565,75 @@ Table 1-2 {productName} Full Profile and Web Profile Features
 |===
 |Feature |Full Platform |Web Profile
 
-|Java Platform, Enterprise Edition 8 |X |X
+|Jakarta EE ${jakartaee} Platform |X |X
 
-|Jakarta EE Security API |X |X
+|Jakarta Security ${jakarta-security-api-version} supporting OpenID Connect |X |X
 
-|Java API for JSON Processing 1.1 |X |X
+|Jakarta JSON Processing ${jakarta-jsonp-api-version} |X |X
 
-|Java API for JSON Binding 1.0 |X |X
+|Jakarta JSON Binding (JSON-B) ${jakarta-json-bind-api-version} |X |X
 
-|EJB 3.2 Lite (Support for Local Container only) |X |X
+|EJB ${jakarta-ejb-api-version} Lite (Support for Local Container only) |X |X
 
-|JTA 1.2 |X |X
+|JTA ${jakarta-transaction-api-version} |X |X
 
 |OSGi-based microkernel |X |X
 
-|Java Web Technologies (Servlet 4.0, JSP 2.2, JSF 2.3, JSTL 1.2, EL 3.0) |X |X
+|Java Web Technologies (Servlet ${jakarta-servlet-api-version}, JSP ${jakarta-pages-api-version}, JSF ${jsf-templating-version}, JSTL ${jstl-api-version}, EL ${jakarta-el-api-version}) |X |X
```

Comment:
`JSF ${jsf-templating-version}`

1. this does not seem to be the right property, as I'd expect something like `jakarta-faces.version`
2. and it's unresolved
<img width="455" height="62" alt="image" src="https://github.com/user-attachments/assets/375b5a25-f146-43d3-ac9d-8ee4b8f5241b" />